conftest.py

```diff
@@ -0,0 +1,6 @@
+import os.path
```

Comment:
You should either change this to just `import os` or change the statements below from `os.path....` to just `path...`

Reply:
I like to import the full module name for clarity 😁